libs/utils/utils.js

```diff
@@ -419,6 +419,8 @@ async function loadPostLCP(config) {
   loadTemplate();
   const { default: loadFonts } = await import('./fonts.js');
   loadFonts(config.locale, loadStyle);
+  const { loadIcons } = await import('./decorate.js');
+  loadIcons();
```

Comment:
What about icons in fragments?  Right now looks like it's only loaded once for `document`

Reply:
@chrischrischris  - I met w/ @auniverseaway yesterday and ironed out a few issues. 
This now works in fragments and follows the same project structure as fragments w/ this functionality living in `features`.